page-components/form-fields.md

```diff
@@ -475,10 +475,7 @@ The action notification displays when something is happening on the page, such a
 {::nomarkdown}
 <div class="m-notification
             m-notification__visible">
-    <span class="m-notification_icon
-                 cf-icon
-                 cf-icon-update
-                 cf-icon__spin"></span>
+    {% include icons/update.svg %}
```

Comment:
Do we want to leave the old, actually-spinning one in place for now?

Reply:
I went back and forth. I can leave it but I was thinking maybe removing it will get us moving to replace it with the SVG animations.